src/Hedgehog/Gen.fs

```diff
@@ -354,7 +368,7 @@ module Gen =
 
     /// Generates a random string.
     let string : Gen<string> =
-        choice [alphaNum; charBounded] |> string'
+        choice [alphaNum; unicodeAll] |> string'
```

Comment:
hmm, I wonder if we should just rename `string'` -> `string` and delete this, let people make their own choices for characters?